app/models/room.py

```diff
@@ -10,4 +10,23 @@ class Room(db.Model):
 
     def __repr__(self):
         return f'<Room {self.name}>'
+    
+class User(db.Model):
```

Comment:
every user should be associated with a specific room

Reply:
should be done